include/inexor/vulkan-renderer/camera.hpp

```diff
@@ -22,11 +31,11 @@ class Camera {
     /// The start position of the camera.
     glm::vec3 m_position{0.0f, 0.0f, 0.0f};
     /// The vector of direction in which the camera is looking.
-    glm::vec3 m_front{0.0f, 0.0f, 1.0f};
+    glm::vec3 m_front{DEFAULT_FRONT};
```

Comment:
Wait! I commited the wrong code changes. This does not compile, it must be `directions::DEFAULT_FRONT`!

Reply:
There I fixed it.